tfx/extensions/google_cloud_ai_platform/training_clients.py

```diff
@@ -423,10 +424,11 @@ def create_training_args(self, input_dict: Dict[Text, List[types.Artifact]],
         {telemetry_utils.LABEL_TFX_EXECUTOR: executor_class_path}):
       job_labels = telemetry_utils.make_labels_dict()
 
-    # 'tfx_YYYYmmddHHMMSS' is the default job display name if not explicitly
-    # specified.
-    job_id = job_id or 'tfx_{}'.format(
-        datetime.datetime.now().strftime('%Y%m%d%H%M%S'))
+    # 'tfx_YYYYmmddHHMMSS_xxxxxxxx' is the default job display name if not
+    # explicitly specified.
+    job_id = job_id or 'tfx_{}_{}'.format(
```

Comment:
Is newly generated job_id logged anywhere later? If not we should use absl.logging.info since the random number is not inferrable elsewhere.